src/BuildingRegistry.Infrastructure/Modules/AggregateSourceModule.cs

```diff
@@ -36,6 +35,8 @@ protected override void Load(ContainerBuilder builder)
                 snapshotStrategy = IntervalStrategy.SnapshotEvery(snapshotInterval);
             }
 
+            builder.RegisterSnapshotModule(_configuration);
```

Comment:
I think we are missing EventHandlingModule, see
https://github.com/Informatievlaanderen/streetname-registry/blob/main/src/StreetNameRegistry.Infrastructure/Modules/AggregateSourceModule.cs#L41

Reply:
line 48